yt/yt/server/controller_agent/controllers/job_info.cpp

```diff
@@ -35,7 +34,6 @@ void TJobNodeDescriptor::RegisterMetadata(auto&& registrar)
     PHOENIX_REGISTER_FIELD(1, Id);
     PHOENIX_REGISTER_FIELD(2, Addresses);
     PHOENIX_REGISTER_FIELD(3, IOWeight);
-    PHOENIX_REGISTER_FIELD(4, Address);
```

Comment:
Let's add a compat comment stating that index 4 is reserved for the deleted field.

Reply:
Added
```
// COMPAT(aleksandr.gaev): index 4 is reserved for deleted field `Address`
```

Shouldn't we write some compat logic which will check snapshot version and will merge Address to Addresses, if it is old enough?
Like we do in TExecNodeDescriptor